src/main/python/afp_cli/compat.py

```diff
@@ -1,6 +1,10 @@
 """ Compatability module for different Python versions. """
 
+
 try:
-    from ordereddict import OrderedDict  # NOQA
+    from ordereddict import OrderedDict
 except ImportError:  # pragma: no cover
-    from collections import OrderedDict  # NOQA
+    from collections import OrderedDict
+
+# Linting purposes, no other use
+OrderedDict
```

Comment:
Let's do a ``# flake8: noqa`` and ``# pragma: no cover`` for the whole file? Then we can remove this.